week-3/slideshow/index.html

```diff
@@ -1,14 +1,32 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
-    <script defer src="slideshow.js"></script>
-  </head>
-  <body>
-    <img id="carousel-img" src="./assets/cute-cat-a.png" alt="cat-pic" />
-    <button type="button" id="backward-btn">Backwards</button>
-    <button type="button" id="forward-btn">Forward</button>
-  </body>
-</html>
+    <title>Image carousel</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <section class="carousel-section">
+        <h2>Image Carousel Level 2</h2>
+        <img id="carousel-img" src="./assets/cute-cat-a.png" alt="cat-pic" /><br>
+        <input type="number" id="inputTime" name="time-input" placeholder="0">
```

Comment:
There is no indication as to what is the purpose of this input field. How could you modify this so that it becomes clear for the user?